setup.py

```diff
@@ -14,32 +14,27 @@
 dependencies = [
     "affine",
     "astroplan",
-    "astropy",
     "backoff",
-    "branca",
     "earthengine-api",
     "earthranger-client",
-    "folium>=0.11.0",
+    "folium",
     "geopandas",
     "igraph",
-    "kaleido",
     "mapclassify",
     "matplotlib",
     "networkx",
     "numba",
-    "numexpr",
-    "numpy",
-    "pandas",
     "plotly",
+    "pre-commit",
     "pyarrow",
-    "pyproj",
```

Comment:
sure we don't need pyproj?

Reply:
I have tested setting up the environment several times over with this setup.py file, and that has not been an issue. That said, I can leave it in, since having it in place doesn't affect the environment.